.github/workflows/metadata-ingestion.yml

```diff
@@ -55,8 +55,13 @@ jobs:
           hadoop-version: "3.2"
       - name: Install dependencies
         run: ./metadata-ingestion/scripts/install_deps.sh
-      - name: Run metadata-ingestion tests (extras ${{ matrix.extraPythonRequirement }})
-        run: ./gradlew -Pextra_pip_requirements='${{ matrix.extraPythonRequirement }}' :metadata-ingestion:${{ matrix.command }}
+      - uses: actions/cache@v3
+        id: cache-venv
+        with:
+          path: ./metadata-ingestion/venv/
+          key: ${{ runner.os }}-venv-${{ hashFiles('./metadata-ingestion/setup.py') }}-${{ matrix.python-version }}-${{ matrix.command }}
```

Comment:
does this need restore keys?

Reply:
More like the other one doesn't need it. Removed it there.

Reply:
If it can find it by key, then that's a good match any other changes causes to rebuild the cache.

Reply:
makes sense